server/bce/utils.py

```diff
@@ -12,28 +13,22 @@ def load_file(file_name, subdirectory="tests/fixtures"):
         return json.loads(data_file.read())
```

Comment:
is this indentation right?

Reply:
yup, (https://docs.python.org/2.5/whatsnew/pep-343.html)